app/assets/stylesheets/show (Alex’s MacBook Air's conflicted copy 2016-01-15).scss

```diff
@@ -0,0 +1,99 @@
+.show-container {
+  border-radius: 5px;
+  background-color: $light-blue;
+  margin: 10px;
+  padding: 5px;
+  position: relative;
+}
+
+.show-title {
+  margin-bottom: 0;
+}
+
+.links {
+  margin-left: 10px;
+  padding-left: 5px;
+
+  p {
+    display: inline;
+    padding-right: 5px;
+  }
+}
+
+.embed-container {
+  overflow: hidden;
+  // position: relative;
+  padding-bottom: 5px;
+  padding-left: 10px;
+  padding-top: 10px;
+  // height: 10px;
+  // width: 75%;
+  // height: 100%;
+//   background-color: $green;
+}
+
+.embed-container iframe,
+.embed-container object,
+.embed-container embed {
+  margin: auto;
+  // position: absolute;
+  // width: 75%;
+  // height: 100%;
+
+}
+
+.voting-title-container {
+  padding-top: 20px;
+  vertical-align: middle;
+}
+
+.voted {
+  font-size: 35px;
+  color: $light-green;
+}
+
+.nil_vote {
+  a {
+  font-size: 25px;
+  }
+}
+
+// .voted {
+//   color: $red;
+//
+//   &:hover {
+//     color: $red;
+//   }
+// }
+
+// .review-container {
+//   float: left;
+// }
+
+.review-title {
+  margin-top: 5px;
+}
+
+
+
+span.voted {
+  padding: 10px;
+}
+
+span.nil_vote {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `nil_vote` should be written in lowercase with hyphens